lib/chef/resource/remote_file.rb

```diff
@@ -94,6 +94,8 @@ def use_conditional_get(true_or_false)
 
       property :show_progress, [ TrueClass, FalseClass ], default: false
 
+      property :ssl_verify_mode, [ :verify_none, :verify_peer, nil ], default: nil
```

Comment:
This should be type Symbol with equal_to set to the values. That way it generates the documentation correctly and still gives nice messages. Also even though the other properties don't have one there should be a description field and an introduced field here (16.1 probably) so we can generate all the docs.

Reply:
Can do. But the overall approach is sane? If so, I'll do that, plus add tests.

Reply:
Also this doesn't need nil. That comes for free

Reply:
the overall approach of this is no less sane than the existing code...